src/webui/src/static/function.ts

```diff
@@ -29,6 +29,56 @@ const convertTime = (num: number): string => {
     }
 };
 
+const convertTimeToSecond = (str: string): number => {
+    const day = str.indexOf('day') !== -1;
```

Comment:
Suggest using `d`, `h`, `m` instead of `day`, `h`, `min`.

Reply:
why do this?

Reply:
We use d/h/m in yaml config file.

Reply:
updated as final comment